ansible-ipi-install/roles/installer/tasks/main.yml

```diff
@@ -25,7 +25,7 @@
   - pullsecret
   - extract
 - include_tasks: 23_rhcos_image_paths.yml
-  when: ((release_version[0]|int == 4) and (release_version[2]|int <= 3)) or cache_enabled|bool
+  when: cache_enabled|bool
```

Comment:
Same here, openshift 4.3 is no longer supported (since Oct 2020), at the very least we'd need to change the version check to 
> (release_version is version("4.3", "<="))

but it seems like we'd be keeping it around for no reason